public/apps/configuration/panels/user-list.tsx

```diff
@@ -213,67 +214,71 @@ export function UserList(props: AppDependencies) {
           <h1>Internal users</h1>
         </EuiTitle>
       </EuiPageHeader>
-      <EuiPageContent>
-        <EuiPageContentHeader>
-          <EuiPageContentHeaderSection>
-            <EuiTitle size="s">
-              <h3>
-                Internal users
-                <span className="panel-header-count">
-                  {' '}
-                  ({Query.execute(query || '', userData).length})
-                </span>
-              </h3>
-            </EuiTitle>
-            <EuiText size="xs" color="subdued">
-              The Security plugin includes an internal user database. Use this database in place of,
-              or in addition to, an external authentication system such as LDAP server or Active
-              Directory. You can map an user account to a role from{' '}
-              <EuiLink href={buildHashUrl(ResourceType.roles)}>Roles</EuiLink>
-              . First, click into the detail page of the role. Then, under “Mapped users”, click
-              “Manage mapping” <ExternalLink href={DocLinks.MapUsersToRolesDoc} />
-            </EuiText>
-          </EuiPageContentHeaderSection>
-          <EuiPageContentHeaderSection>
-            <EuiFlexGroup>
-              <EuiFlexItem>{actionsMenu}</EuiFlexItem>
-              <EuiFlexItem>
-                <EuiButton
-                  fill
-                  href={buildHashUrl(ResourceType.users, Action.create)}
-                  data-test-subj="create-user"
-                >
-                  Create user account
-                </EuiButton>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPageContentHeaderSection>
-        </EuiPageContentHeader>
-        <EuiPageBody>
-          <EuiInMemoryTable
-            tableLayout={'auto'}
-            loading={userData === [] && !errorFlag}
-            columns={getColumns(currentUsername)}
-            // @ts-ignore
-            items={userData}
-            itemId={'username'}
-            pagination
-            search={{
-              box: { placeholder: 'Search internal users' },
-              onChange: (arg) => {
-                setQuery(arg.query);
-                return true;
-              },
-            }}
-            // @ts-ignore
-            selection={{ onSelectionChange: setSelection }}
-            sorting
-            error={errorFlag ? 'Load data failed, please check console log for more detail.' : ''}
-            message={showTableStatusMessage(loading, userData)}
-          />
-        </EuiPageBody>
-        {deleteConfirmModal}
-      </EuiPageContent>
+      {errorFlag ? (
+        <AccessErrorComponent dataSourceLabel={dataSource && dataSource.label} />
```

Comment:
I am not sure if we can always assume that the error flag means MDS issue, it could also be network issue/whatever was the list of issues possible before MDS was enabled WDYT?

Reply:
Addressed.